index.js

```diff
@@ -91,33 +91,56 @@ const stonksifyRules = rules.sort(function compare(a, b) {
   }
 });
 
-input.on("keyup", function (e) {
+input.on("keyup", function(e) {
   if (e.which === 13) {
     console.log("Hit Enter");
     enterWord(input.val());
   }
 });
 
-submitButton.on("click", function () {
+submitButton.on("click", function() {
   console.log("Clicked Submit");
   enterWord(input.val());
 });
 
-shareButton.on("click", function () {
+shareButton.on("click", function() {
   copyToClipboard();
-  shareButton.text("Copied to Clipboard!");
+  // adds a span to display "Link Copied"
+  // then, styles the text above the input field's position and animates the text to float upward before removing from the dom
```

Comment:
Instead of these comments, i'd just suggest a function with a descriptive name like "showLinkCopiedPopUpText". Much more succinct.

Reply:
I'd have to move all this into a new function to do that. Seems more complex than leaving it in here with a comment